backend/src/main/java/com/step2020/server/common/Event.java

```diff
@@ -18,30 +18,39 @@
   
   private String eventId;
   private String eventName;
+  private String date;
+  private String startTime;
+  private String endTime;
   private String description;
   private String location;
   private String ownerId;
   private String organization;
-  private String imagePath;
+  private String imagePaths;
 
   public static class Builder {
 
     private String eventId;
     private String eventName;
+    private String date;
+    private String startTime;
+    private String endTime;
     private String description;
     private String location;
     private String ownerId;
     private String organization;
-    private String imagePath;
+    private String imagePaths;
 
     public Builder() {
       this.eventId = "";
       this.eventName = "";
+      this.date = "";
+      this.startTime = "";
+      this.endTime = "";
       this.description = "";
       this.location = "";
       this.ownerId = "";
       this.organization = "";
-      this.imagePath = "";
+      this.imagePaths = "[]";
```

Comment:
I'm a little bit confused about the value of imagePaths. because imagePaths is set to literal [ ], i don't know if you meant an array or not.